app/src/components/grids/cardsGrid.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import React from "react";
+import useBreakpoints from "../../hooks/useBreakpoints";
+import CollectionDataType from "../../types/CollectionDataType";
+import { CollectionCardModel } from "../../models/collectionCard";
+import ItemDataType from "@/src/types/ItemDataType";
+import ItemCardDataType from "@/src/types/ItemCardDataType";
+import { ItemCardModel } from "@/src/models/itemCard";
+import { SimpleGrid as DCSimpleGrid } from "../simpleGrid/simpleGrid";
+import { Card as DCCard } from "../card/card";
+
+interface CardsGridProps {
+  records: CollectionDataType[] | ItemDataType[];
+}
+
+function isCollectionType(
+  records: CollectionDataType[] | ItemDataType[]
+): records is CollectionDataType[] {
+  return "numberOfDigitizedItems" in records[0];
+}
+
+export const CardsGrid = ({ records }: CardsGridProps) => {
+  const { isLargerThanLargeTablet } = useBreakpoints();
+  const isCollections = isCollectionType(records);
+
+  return (
+    <DCSimpleGrid>
+      {records?.map((record, index) => {
+        if (isCollections) {
+          const collectionCardModel = new CollectionCardModel(record);
+          return (
+            <DCCard
+              key={index}
+              id={index}
+              slug={collectionCardModel.title}
+              record={collectionCardModel}
+              isLargerThanLargeTablet={isLargerThanLargeTablet}
```

Comment:
Do these `DCCard`s need the `ref` and `tooltipOffset` props?

Reply:
ah, good catch. I think so.

Reply:
I only think I need to add this to the collection cards for now, not the item cards?

Reply:
Both need it